lib/stack_master/commands/lint.rb

```diff
@@ -0,0 +1,43 @@
+require 'tempfile'
+
+module StackMaster
+  module Commands
+    class Lint
+      include Command
+      include Commander::UI
+
+      def initialize(config, stack_definition, options = {})
+        @config = config
+        @stack_definition = stack_definition
+      end
+
+      def perform
+        unless cfn_lint_available
+          puts "Failed to run cfn-lint, do you have it installed and available in $PATH?"
+          return
+        end
+
+        Tempfile.open(['stack', ".#{proposed_stack.template_format}"]) do |f|
+          f.write(proposed_stack.template_body)
+          f.flush
+          system('cfn-lint', f.path).nil?
+          puts "cfn-lint run complete"
```

Comment:
Can we pass the exit code from cfn-lint back?

Reply:
cf-lint doesn't care :( it's always 0, whether it fails or not.

Reply:
Boo